src/test/kotlin/com/terraformation/backend/accelerator/db/VoteStoreTest.kt

```diff
@@ -566,5 +634,84 @@ class VoteStoreTest : DatabaseTest(), RunsAsUser {
         store.upsert(projectId, phase, newUser, null, null)
       }
     }
+
+    @Test
+    fun `computes vote decision on creating one vote`() {
```

Comment:
I think the common scenario will be that voters are added with no vote initially (maybe populated from the default voters list). Might be worth testing that when a voter with no vote is added, the decision ends up null.